code-challenges.js

```diff
@@ -54,7 +128,32 @@ var notAString2 = 42
 
 var toonimals = [ {name: "Itchy", animal: "mouse"}, {name: "Stimpy", animal: "cat"}, {name: "Daffy", animal: "duck"}, {name: "Scratchy", animal: "cat"}, {name: "Ren", animal: "dog"}, {name: "Felix", animal: "cat"}]
 
+// Declare function that takes in an array as an argument
+const catsOnly = (arr) => {
+  // Filter through array using value
+  return arr.filter(value => {
+    // Return only index values where value.animal equals "cat"
+    return value.animal === "cat"
+  })
+}
```

Comment:
amazing! if you _wanted_ to make this all into one line of code you can do that as well! ill give you an example below but this solution totally works too! just an option :)

```javascript
const catsOnly = (arr) => {
  // Filter through array using value
  return arr.filter(value => value.animal === "cat")
}
```